lib/license_finder/reports/text_report.rb

```diff
@@ -2,8 +2,14 @@ module LicenseFinder
   class TextReport < CsvReport
     COMMA_SEP =  ", "
 
-    def initialize(dependencies, options={})
-      super(dependencies, options.merge(columns: %w[name version licenses]))
+    def initialize(dependencies, options)
+      empty_options = {}
+      default_columns = %w[name version licenses]
+      columns = (Array(options[:columns]) & self.class::AVAILABLE_COLUMNS)
+      if columns.empty?
+        columns = default_columns
+      end
+      super(dependencies, empty_options.merge(columns: columns))
```

Comment:
Thank you for the review.

As you say, using the `:columns` inherited from `CsvReport` is safer and simpler.

I tried correcting it. Can you review it again?